### ISSUE TITLE
[PROPOSAL] Add keyframes to output

### ISSUE THREAD

**Issue by nickcmaynard** [nishad/udemy-dl#167]
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION].

Replace everything between stars with current version of your udemy-dl (output of udemy-dl -v):  
Version : **0.2.2-alpha**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [X] I have read and understood the [contributors guide](https://github.com/nishad/udemy-dl/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed issues](https://github.com/nishad/udemy-dl/issues?q=is%3Aissue+is%3Aclosed) and in [opened issues](https://github.com/nishad/udemy-dl/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never used the project.
- [X] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

**If the project maintainer has any additional requirements, they will be listed here.**

- No additional requirements.

---

Downloaded (one assumes assembled) videos have no keyframes, and consequently seeking within them (in VLC at least) is broken.  I propose the assembly process inserts keyframes at an appropriate (frequent!) interval.

**Comment by nishad** [comment]
Is the result same if you use `ffmpeg` instead of builtin downloader with `--use-ffmpeg` option ?

**Comment by nickcmaynard** [comment]
You're absolutely right - that does indeed fix things for me.  I wonder - should "good' videos be default behaviour?
